packages/gatsby-cli/src/reporter/loggers/json/index.ts

```diff
@@ -0,0 +1,28 @@
+import { onLogAction } from "../../redux/index"
+import { ActionsUnion, ISetStatus } from "../../redux/types"
+import stripAnsi from "strip-ansi"
+import _ from "lodash"
+
+const isStringPayload = (action: ActionsUnion): action is ISetStatus =>
```

Comment:
can we get away without this `action is ISetStatus` ?

Reply:
I use `action is ISetStatus` to filter out ISetStatus which payload is a string.